symbolic-ppdb/src/format/mod.rs

```diff
@@ -231,6 +231,9 @@ impl<'data> PortablePdb<'data> {
             guid_stream: None,
         };
 
+        // Collect the streams into a map, because they can't necessarily
+        // be parsed in the order they're encountered.
+        let mut streams = HashMap::new();
```

Comment:
This might as well be a `Vec`. Or maybe even a bunch of hardcoded `Option<...>` values, as we known exactly which streams we expect / can use. Then we can also process them in a deterministic order (as hashmap iteration order is still random).

Reply:
Now that I think about it, an even simpler way to do it would be to parse them in any order, but always do `#~` last.